e2e/utils.go

```diff
@@ -265,7 +265,7 @@ func getStorageClass(path string) scv1.StorageClass {
 // 	return sc
 // }
 
-func createCephfsStorageClass(c kubernetes.Interface, f *framework.Framework, enablePool bool) {
+func createCephfsStorageClass(c kubernetes.Interface, f *framework.Framework, enablePool bool, parameters map[string]string) {
```

Comment:
cleanup: how about dropping `enablePool` and setting it in the `parameters` instead?